ansys/dpf/core/plotter.py

```diff
@@ -280,14 +381,14 @@ def plot_chart(self, fields_container):
         return pyplot.legend()
 
     def plot_contour(
-        self,
-        field_or_fields_container,
-        notebook=None,
-        shell_layers=None,
-        off_screen=None,
-        show_axes=True,
-        meshed_region=None,
-        **kwargs
+            self,
+            field_or_fields_container,
+            notebook=None,
+            shell_layers=None,
+            off_screen=None,
+            show_axes=True,
+            meshed_region=None,
+            **kwargs
```

Comment:
@akaszynski, @anslpa : I use PyCharm, the auto pep8 does some inconsistent formatting changes, how do you tackle these?

Reply:
We should be using `black` in a pre-commit as in https://github.com/ansys/aedt-testing/blob/master/.github/workflows/pre-commit.yml to avoid any inconsistencies with competing auto-formatters. I'm fine with these whitespace changes, but we really should have a consistent code format.

Reply:
Thanks @akaszynski, I will set this up at my end. I totally agree, we should have a consistent code format, it makes reviewing changes a less painful process.